packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -235,6 +235,10 @@ const NuovaDelega = () => {
                                 label={t('nuovaDelega.form.naturalPerson')}
                               />
                               <FormControlLabel
+                                onClick={() => {
```

Comment:
The setFieldValue method run validation by default. Disable validation by adding a third parameter set to false.
We also must reset touched property. For this purpose create an async - await function that uses setFieldValue (with await, because it returns a promise) and setFieldTouched (with await)

Reply:
fix it

Reply:
I found (against my previous idea) that in fact neither `setFieldValue` nor `setFieldTouched` are async in the way we are using Formik
![image](https://user-images.githubusercontent.com/5519535/210632530-47ec2d58-b6b5-46fb-b6b9-c09f1423a8a7.png)

Reply:
in this two function, when called, not are used with async/await

Reply:
The issue about multiple validation runs has been solved properly.